src/main/java/com/thebois/models/beings/AbstractBeing.java

```diff
@@ -22,7 +24,10 @@
 
     // The max speed of the AbstractBeing
     private static final float MAX_WALKING_DISTANCE = 0.1f;
+    // How many kilograms a being can carry.
+    private static final float MAX_CARRYING_CAPACITY = 100f;
```

Comment:
Guessing it checkstyle complained about magic numbers? Or why would we want it like this?

Reply:
>Guessing it checkstyle complained about magic numbers? 

Nope, but it might've done so otherwise, I don't know. I wrote it like this directly.

Constants should always be *actual constants*, defined at the top of the class with a clear name. 

>Or why would we want it like this?

Why *wouldn't* we? Would you have preferred
```java
private IInventory inventory = new Inventory(100f);`
```
?

If I just write `100f` then that could mean anything, and I'd have to look at the constructor documentation to see what it means. But now that it says
```java
private IInventory inventory = new Inventory(MAX_CARRYING_CAPACITY);
```
, you can clearly understand what the argument means.

It's like with `MAX_WALKING_DISTANCE` right above. That would be really unclear if it was just a constant in the code.